src/fe-small-advertisements/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Perhaps you could use our project name here

Reply:
Thanks, yes. You are totally right with all those comments. It is ok if now I just change the name and delete the section with the unnecessary react information but don't make the personalize the complete readme? I would add a sentence that I could improve later. With some good templates I have saved.

Reply:
Yes thats fine. Can you make an issue for that on the board so we can keep track of it?

Reply:
Yes, I will.